tests/integration/TestSuite.php

```diff
@@ -1,4 +1,5 @@
 <?php
```

Comment:
Heh, yea this PR is for the 1.4.x client, addressing an issue for someone who still uses it. I should probably stop making updates to it, but since people are using it, I feel somewhat obligated. ;)